components/release.sh

```diff
@@ -81,6 +81,19 @@ git add --all
 git commit --message "Updated component images to version $COMMIT_SHA"
 image_update_commit_sha=$(git rev-parse HEAD)
 
+# Updating the samples to use the updated components
+git diff HEAD~1 HEAD --name-only | while read component_file; do
+    echo $component_file
+    find components samples -type f | while read file; do
+      sed -i -E "s|(https://raw.githubusercontent.com/kubeflow/pipelines/)[^/]+(/$component_file)|\1${image_update_commit_sha}\2|g" "$file";
```

Comment:
Are you sure this commit sha will be the same after the PR is merged into master?

Reply:
> Are you sure this commit sha will be the same after the PR is merged into master?

The original commit remains accessible even if the branch was merged and deleted.

Reply:
Thanks.

/lgtm